### PR TITLE
Upload code to reproduce paper results

### PR DESCRIPTION
**Pull request by FabioBergonti**
This PR uploads the code and a first version of the documentation for reproducing paper results.

---

**Known issues** that I have to fix in the next PRs:
- [x] upload the missing image `images/sim3.png`
- [x] fill ********* with time 
https://github.com/ami-iit/paper_bergonti_2022_tro_kinematics-control-morphingcovers/blob/bd56a76055e8ec069d230947c863ebf5c6702b61/scripts/sim3.m#L24
- [x] fix paper links in the readme
- [x] create a release in mystica and checkout to that specific tag
- [x] export conda list with version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+## Installation
+
+1. Clone [`this`](https://github.com/ami-iit/paper_bergonti_2022_tro_kinematics-control-morphingcovers) repository  and [`mystica`](https://github.com/ami-iit/mystica/):
+  ```bash
+  git clone https://github.com/ami-iit/paper_bergonti_2022_tro_kinematics-control-morphingcovers.git
+  git clone https://github.com/ami-iit/mystica.git
+  ```
+  [mystica](https://github.com/ami-iit/mystica/) is a matlab library to simulate the kinematics and dynamics of multibody systems.
+
+2. Run in **matlab** the function [`install()`](https://github.com/ami-iit/mystica/blob/main/install.m) stored in [`mystica`](https://github.com/ami-iit/mystica/):
```
Comment (gabrielenava) [inline README.md:43]: are there limitation on the MATLAB version one can use?

File: src/motor-positioning/@GeneticAlgorithm/updateRouletteWheel.m [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function updateRouletteWheel(obj)
```
Comment (gabrielenava) [inline src/motor-positioning/@GeneticAlgorithm/updateRouletteWheel.m:1]: roulette wheel? :-D

File: README.md [hunk 3]
```diff
@@ -1 +1,98 @@
...
+  ``` matlab
+  install('env_name','imorph')
+  ```
+The function [`install()`](https://github.com/ami-iit/mystica/blob/main/install.m) downloads [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge). [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge) is a package manager that downloads and configures our dependencies in conda enviroment called `imorph`.
+
+## Usage
+
+This repository stores:
+- the algorithm for evaluating an optimal motor positioning for structures with closed kinematic loops (see Sec.IV of the [paper](paper_link_to_do));
```
Comment (gabrielenava) [inline README.md:52]: `[paper](paper_link_to_do)`  is it a reminder or you forgot to update the link? Maybe with the arxiv version for now

**Review by gabrielenava — APPROVED** [review]
approving with minor comments, I am assuming that this review does not require for now also the testing of the repo, that will be performed later

File: README.md (hunk 1 above)
Comment (FabioBergonti) [reply README.md:43]: The project works with the Matlab basic license. Simulink and all the other toolboxes are not required.
However, I don't know if it works for all downloadable releases.
I tested it successfully on Matlab R2022a, R2020b, and R2019b

Comment (traversaro) [reply README.md:43]: Just fyi (not something that needs to be done), if you have some tests that can easily run you can use the MATLAB support for GitHub Actions to easily test all the releases, as done for example in https://github.com/robotology/urdf2casadi-matlab/blob/master/.github/workflows/matlab_ci.yml .

**Review by traversaro — APPROVED** [review]

File: src/motor-positioning/@GeneticAlgorithm/updateRouletteWheel.m (hunk 2 above)
Comment (FabioBergonti) [reply src/motor-positioning/@GeneticAlgorithm/updateRouletteWheel.m]: Fixed in https://github.com/ami-iit/paper_bergonti_2022_tro_kinematics-control-morphingcovers/commit/f8b61d5d2560a1af5356fa5d3b4de5acbe74a349

File: README.md (hunk 3 above)
Comment (FabioBergonti) [reply README.md:52]: it's a reminder :)
Both the IEEE explore and arxiv version are not yet available

**Comment by FabioBergonti** [comment]
Thanks @gabrielenava and @traversaro for the review

**Pull request merged**